src/utils/createDynamicAppAtScriptTag.ts

```diff
@@ -70,6 +70,14 @@ export const createDynamicAppAtScriptTag = async (
     scriptEl.insertAdjacentElement('afterend', mountTargetEl)
   }
 
+  if (
+    !isDevelopment &&
+    (props.productPrice === undefined || props.merchantId === undefined)
```

Comment:
Is it correct to just look for explicit undefined here? Should you instead just check for `!props.productPrice` etc to catch more cases?